### ISSUE TITLE
fs.existsSync returns true for filename

### ISSUE THREAD

**Issue by HarshitJoshi** [jliu2179/teleporter#5]
There's an issue with the current implementation of adding locations to the database and teleporting to them. `fs.existsSync` module returns true for directories as well as filenames. When you try to add files as your teleport location, it successfully adds it to the database. Later, followed by no action when using `tp to location` command. 
```
tp -a testing ~/Documents/someFile.txt

tp -l
Locations stored in DB:
-----------------------
testing ==> /home/Documents/someFile.txt

tp to testing (nothing, still at the same directory)

```
@jliu2179, I believe you should add another check for directory vs other things before saving to the db. `lstat` function in the `fs` module has a .isDirectory() method that would come in handy.

**Comment by cliuj** [comment]
Nice catch. I'll look into it and release a fix later today.

**Comment by cliuj** [comment]
Issue fixed in commit: 5dad0d232401067f88b414b01776afde35779444. Closing this issue.
